src/molecules/Bar.tsx

```diff
@@ -28,7 +29,9 @@ export const Bar: React.FC = () => {
       <Box marginHorizontal="20px">
         <PersonIcon size="35" sizeMd="22" />
       </Box>
-      <CartIcon size="35" sizeMd="22" />
+      <Link to="/cart">
```

Comment:
**Tip:** you can keep your routes in a separate file. That way, if a route change you would change it only in one file, and all the other places would reference that value.

Consider this:

```ts
// src/routes.ts
export default {
  home: '/',
  cart: '/cart',
  productDetail: '/product/:productId'
}
```

```tsx
// src/App.tsx
// Use the routes in your router
<Route path={routes.cart} component={CartPage} />
```

```tsx
// src/molecules/Bar.tsx
import routes from '../routes'

// Reference the routes when you need to navigate to them
<Link to={routes.cart}>{...}</Link>
```